test/framework/options.go

```diff
@@ -51,6 +53,9 @@ func (options *Options) Validate() error {
 	if len(options.AWSVPCID) == 0 {
 		return errors.Errorf("%s must be set!", "aws-vpc-id")
 	}
+	if len(options.HelmDir) == 0 {
```

Comment:
Comment from PR 2433, 

@kishorj  2 hours ago

why not make it optional? If set, test uses the specified directory, otherwise default to the aws eks-charts repo.


@johngmyers  1 hour ago

Making it optional adds complexity for no obvious reason and admits the possibility of missing an incompatibility between the repo's chart and the repo's image.

Could you give more information on the use case for this? Why would one not want to put the repo's Helm chart under test?

@kishorj  now

One example is the canary script which we run against the release version. In this case we want the last published image and the chart, not the recent one from the repo.

Reply:
One possibility would be to download the published chart in the script, outside of ginkgo. But it might work to rely on the scripts' current `helm repo add` and pass the name of the chart.